### DISCUSSION TITLE
Jetson Nano Input Voltage - 5.0V or 5.4V?

### DISCUSSION THREAD

**Discussion by brightsnowzhao** [prl-mushr/mushr#66] [Hardware]
Hi, my Jetson Nano would not boot when connected to the buck convertor that is tuned to 5.0V.  But it works while on the 5V power supply.  I measured the voltage and found that the convertor out put was exactly 5V, but the power supply outputs 5.4V.  I tried tuning the converter to 5.4V and it started to work.

Did you see such phenomenon with other users?

**Comment by palanc** [comment]
Hi @brightsnowzhao - I'm glad that you were able to solve that issue. Yes we have seen this before, more details here: [https://github.com/prl-mushr/mushr/discussions/51](https://github.com/prl-mushr/mushr/discussions/51)

Let us know if you have any other questions!

**Reply by brightsnowzhao** [reply]
Thank you!
